src/main/java/spoon/reflect/declaration/CtElement.java

```diff
@@ -35,7 +36,7 @@
  * element).
  */
 @Root
-public interface CtElement extends FactoryAccessor, CtVisitable, Cloneable {
+public interface CtElement extends FactoryAccessor, CtVisitable, Cloneable, QueryComposer {
```

Comment:
QueryComposer contains all the methods, which can begin with creation of some query. It is the replacement for `CtElement#query()`. With QueryComposer you can easily start to query spoon model starting from any element. 
Note: QueryComposer is implemented by QueryStep too.